src/ng-backstretch.js

```diff
@@ -21,9 +21,9 @@ directive('backstretch', ['$window', '$timeout', function($window, $timeout) {
        * e.g. <div backstretch backstretch-url="'/path/to/image.jpg'">
        * So, we need to turn this back into an array.
        */
-      scope.images = Array.isArray(scope.images()) ? scope.images() : [scope.images()];
-      scope.duration = scope.duration() || 5000;
-      scope.fade = scope.fade() || 1;
+      scope.images = Array.isArray(scope.images) ? scope.images : [scope.images];
+      scope.duration = !angular.isUndefined(scope.duration) ? scope.duration : 5000;
```

Comment:
I'm new to Angular, I think you're right on the bindings.  However, when trying to set ```fade``` to ```0``` to disable it, ```0 || 1``` results in ```1``` which sets ```fade``` to ```1```.  Are your demo pics from Garfield Park in Chicago?